app/models/agency.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+# Agencies provide child care subsidy payments
+class Agency < ApplicationRecord
+  # Handles UUIDs breaking ActiveRecord's usual ".first" and ".last" behavior
+  self.implicit_order_column = 'created_at'
+
+  validates :active, inclusion: { in: [true, false] }
+  validates :name, presence: true
```

Comment:
State is also required!

```suggestion
  validates :name, presence: true
  validates :state, presence: true
```